modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -4,27 +4,19 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cluster, deploy
 
-*_Enterprise Edition_*
-
 This page describes manual creation and configuration of a cluster. There are two ways to create and manage clusters:
 
 * Using Runtime Manager
 +
 See xref:runtime-manager::managing-servers.adoc#create-a-cluster[Creating and Managing Clusters] for configuration instructions.
 * Manually, using a configuration file
 
+== Considerations
```

Comment:
Can we use a more precise title?
How about Cluster Requirements and Restrictions?